docs/gettingstarted/index.rst

```diff
@@ -133,7 +133,7 @@ Getting started with Flocker
 
          .. code-block:: console
 
-            you@laptop:~$ sudo apt-get update && apt-get install -y gcc python2.7 python-virtualenv python2.7-dev && \
+            you@laptop:~$ sudo apt-get update && sudo apt-get install -y gcc libffi-dev libssl-dev python2.7 python-virtualenv python2.7-dev && \
```

Comment:
I've seen failures related to these not being installed.  But oh well, maybe they won't bite anyone else (it would have been safer just to leave these even if they're not necessary, though).